libraries/opensk/src/api/crypto/rust_crypto.rs

```diff
@@ -205,7 +205,6 @@ impl ecdsa::Signature for SoftwareEcdsaSignature {
         Some(SoftwareEcdsaSignature { signature })
     }
 
-    #[cfg(feature = "std")]
```

Comment:
I guess this was actually used when combining `std` and `rust-crypto`?

Reply:
We needed this function in another Env (outside of `std`), so it had to become part of the API. That was already done for `software_crypto`, but I forgot for `rust_crypto`. Our new checks cover more, so this was caught.